### PR TITLE
Update path

### PR DESCRIPTION
**Pull request by ShalokShalom**
I corrected the path to your mdx. 👍🏼 

It changed along the path of the development and got not updated along with it. 

There are several projects, who check for dead links. 

https://github.com/MrCull/GitHub-Repo-ReadMe-Dead-Link-Finder 

Thanks

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [] Added new tests to cover the fix / functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
 
-Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable.mdx).
+Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable/index.mdx)
 
 As users interact with the site, only the necesary parts of the site load on-demand. This [precision lazy-loading](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/progressive.mdx) is what makes Qwik so quick.
 
 ## Getting Started
 
-Try out our starter:
+Try out our starter: it
```
Comment (adamdbradley) [inline README.md:19]: Could the `it` be removed here?

File: README.md [hunk 2]
```diff
@@ -10,13 +10,13 @@
 
 # The HTML-first framework
 
-Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable.mdx).
+Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable/index.mdx)
```
Comment (adamdbradley) [inline README.md:13]: I think this paragraph was written before we had a docs site. Would you be able to point the url to https://qwik.builder.io/docs/concepts/resumable instead?

Comment (ShalokShalom) [reply README.md]: Both resolved

File: README.md (hunk 1 above)
Comment (ShalokShalom) [reply README.md]: Done

**Comment by shairez** [comment]
Thanks @ShalokShalom !

It looks like this change got merged on a different PR
Thank you for taking the time to create these changes, we appreciate your work on this! 🙏

**Pull request closed without merging**